sdk/ml/azure-ai-ml/azure/ai/ml/entities/_manage_clouds.py

```diff
@@ -0,0 +1,7 @@
+from azure.ai.ml._azure_environments import (_add_cloud, _update_cloud)
+
+def add_cloud(cloud):
```

Comment:
Please add the docstring for this publicly exposed method so that reference documentation can be generated automatically from the docstrings.